Projects/Claims/Sources/Views/submitClaim/Checkout/SubmitClaimCheckoutTransferringScreen.swift

```diff
@@ -3,169 +3,238 @@ import hCore
+                    .foregroundColor(hLabelColor.primary)
+                hText(error, style: .footnote)
+                    .multilineTextAlignment(.center)
+                    .foregroundColor(hLabelColor.primary)
+                    .matchedGeometryEffect(id: "titleLabel", in: animation)
             }
-            .ignoresSafeArea()
-            .frame(maxWidth: .infinity, maxHeight: .infinity)
-            ZStack(alignment: .center) {
-                content()
-                    .padding(.horizontal, 24)
+            .scaleEffect(
+                x: errorAnimation ? 1 : 0.3,
+                y: errorAnimation ? 1 : 0.3,
+                anchor: .center
+            )
+
+            Spacer()
+
+            hButton.LargeButtonFilled {
+                store.send(.dissmissNewClaimFlow)
+                store.send(.openFreeTextChat)
+            } content: {
+                hText(L10n.openChat)
             }
-            .frame(maxWidth: .infinity, maxHeight: .infinity)
-            .onAppear {
-                withAnimation {
-                    isAnimating = true
+            HStack {
+                hButton.LargeButtonText {
+                    store.send(.dissmissNewClaimFlow)
+                } content: {
+                    hText(L10n.generalCloseButton)
+                }
+
+                hButton.LargeButtonText {
+                    store.send(.claimNextSingleItemCheckout)
+                } content: {
+                    hText(L10n.generalRetry)
                 }
             }
-            .onChange(of: colorScheme) { newValue in
+        }
+        .opacity(errorAnimation ? 1 : 0)
+        .disabled(!errorAnimation)
+        .animation(
+            .interpolatingSpring(
+                stiffness: 170,
+                damping: 15
+            )
+            .delay(0.25),
+            value: errorAnimation
+        )
+        .onAppear {
+            if !errorAnimation {
                 withAnimation {
-                    isAnimating = false
-                }
-
-                Task {
-                    await delay(0.25)
-                    withAnimation {
-                        isAnimating = true
-                    }
+                    errorAnimation = true
                 }
             }
         }
+
+    }
+
+    func checkIfNotDecimal(value: Double) -> Bool {
+        if value.truncatingRemainder(dividingBy: 1) == 0 {
+            return true
+        }
+        return false
     }
 
+    func formatDoubleWithoutDecimal(value: Double) -> String {
+        let formatter = NumberFormatter()
+        formatter.groupingSeparator = " "
+        formatter.numberStyle = .decimal
+        formatter.minimumFractionDigits = 0
+        formatter.maximumFractionDigits = 0
+        return formatter.string(for: value) ?? ""
+    }
+
+    func formatDoubleWithDecimal(value: Double) -> String {
+        let formatter = NumberFormatter()
+        formatter.groupingSeparator = " "
```

Comment:
Should we override this? isn't this locale dependant, i think just setting locale on the formatter should solve this.